MapboxNavigation/NavigationViewController.swift

```diff
@@ -124,8 +124,10 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
      on the destination of your route. The last coordinate of the route will be
      used if no destination is given.
     */
+    @available(*, deprecated, message: "Destination is no longer support nor necessary. A destination annotation will automatically be added to map given the route.")
```

Comment:
@1ec5 it was actually added in https://github.com/mapbox/mapbox-navigation-ios/pull/268